jena-permissions/pom.xml

```diff
@@ -19,9 +19,7 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.apache.jena</groupId>
   <artifactId>jena-permissions</artifactId>
-  <version>3.6.0-SNAPSHOT</version>
```

Comment:
This isn't "Upgrading minor dependencies".

IIRC Having the version in the module is, (or was - maven changes) necessary so that modules can be built on their own during development.

This is the style elsewhere in the project - one change and not others seems odd.